packages/shared/friends/utils.ts

```diff
@@ -132,3 +133,23 @@ export function validateFriendRequestId(friendRequestId: string, ownId: string)
 
   return friendRequestId.includes(ownId)
 }
+
+/**
+ * Check whether a user is blocked by the current user
+ * @param userId - the ID of the user to check
+ * @returns true if the user is blocked, false otherwise
+ */
+export function isBlocked(userId: string) {
+  // Get the user's profile
+  const profile = getCurrentUserProfile(store.getState())
+
+  // Check whether the user is blocked
+  if (profile?.blocked?.includes(userId)) {
+    return true
+  }
+  return false
+}
+
+export const DEFAULT_MAX_NUMBER_OF_REQUESTS = 5
+
+export const COOLDOWN_TIME = 1000 // 1 second
```

Comment:
Make constant name imply time unit.
```suggestion
export const COOLDOWN_TIME_MS = 1000 // 1 second
```